country_codes.js

```diff
@@ -0,0 +1,287 @@
+const EUCountries = {
+  Austria:      "AT",
```

Comment:
oh...usually it's the other way around, like `AT: 'Austria'`. It's a key-value pair and in this situation country code should be key. 
And below you can get country codes easier.

Reply:
Ok. I change it.